.github/workflows/ci_cd.yml

```diff
@@ -35,54 +36,14 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
   doc-build:
-    name: Build Documentation
+    name: Documentation build
     runs-on: ubuntu-latest
-    needs: doc-style
-
     steps:
-      - name: "Checkout the repository"
-        uses: actions/checkout@v4
-
-      - name: "Set up Python"
-        uses: actions/setup-python@v5
+      - name: "Run Ansys documentation building action"
+        uses: ansys/actions/doc-build@v5
         with:
-          python-version: ${{ env.MAIN_PYTHON_VERSION }}
-
-      - name: "Install system dependencies"
-        run: |
-          sudo apt-get update && sudo apt-get install pandoc libkrb5-dev
-
-      - name: "Install Python dependencies"
-        run: |
-          python -m pip install --upgrade pip tox
-          python -m pip install poetry~=1.7.0
-
-      - name: Install library
-        run: |
-          poetry install --with docs --extras "oidc linux-kerberos"
-
-      - name: Build HTML
-        run: make -C doc html SPHINXOPTS="-W"
-
-      - name: Build PDF Documentation
-        run: |
-          sudo apt update
-          sudo apt-get install -y texlive-latex-extra latexmk texlive-xetex fonts-freefont-otf xindy
-          make -C doc latexpdf
-
-      - name: Upload HTML Documentation
-        uses: actions/upload-artifact@v4
-        with:
-          name: documentation-html
-          path: doc/build/html
-          retention-days: 7
-
-      - name: Upload PDF Documentation
-        uses: actions/upload-artifact@v4
-        with:
-          name: documentation-pdf
-          path: doc/build/latex/*.pdf
-          retention-days: 7
+          check-links: false
+          sphinxopts: "-j 1 -W --keep-going"
```

Comment:
I realized after adding `-j 1` here that the `-j auto` default was actually coming from the Makefile, so the better option is to delete the SPHINXOPTS values in the Makefile, instead of defining `-j 1` here.

Shouldn't be changed in this PR, we should change it in main if we're going to do it anywhere. It's completely optional though, just thought I'd mention it.

Reply:
I'll open a PR on main. It shouldn't matter on this repo, because the docs are quite simple, but let's remove unnecessary noise